lasagne/tests/test_utils.py

```diff
@@ -182,7 +213,7 @@ def test_unroll_scan():
     b = T.scalar("b")
 
     def mul_div(step, previous_mul, previous_div, mul, div):
-            return previous_mul*mul, previous_div/div
+        return previous_mul * mul, previous_div / div
```

Comment:
Same here. Even if the indentation was funny, we shouldn't mix formatting fixes with implementations of new features.